backend/fittoring/src/main/java/fittoring/application/auth/presentation/AuthController.java

```diff
@@ -50,7 +55,7 @@ public class AuthController {
 
     @PostMapping("/signup")
     public ResponseEntity<Void> signUp(@RequestBody @Valid SignUpRequest request) {
-        authService.register(request);
+        authService.register(request.loginId(), request.name(), request.gender(), request.phoneNumber(), request.password());
```

Comment:
p2: 타입 안정성을 위해서 `phoneNumber`를 포장해서 넘겨주면 어떨까요?

```suggestion
        authService.register(request.loginId(), request.name(), request.gender(), new Phone(request.phoneNumber()), request.password());
```

현재 컨트롤러에서 `phoneNumber`를 원시 타입으로 넘겨주는 모든 부분에도 해당하는 코멘트 입니다!